docs/spec/spec.md

```diff
@@ -522,7 +522,7 @@ service "HelloWorld" on new grpc:Listener(9090) {
 }
 ```
 
-#### 5.1.1.3 Service - JWT Auth
+#### 5.1.1.3 Service - JWT auth
```

Comment:
```suggestion
#### 5.1.1.3 Service - JWT Auth
```
Check how the industrial term is written.

Reply:
https://jwt.io/introduction